types/ssh2-sftp-client/index.d.ts

```diff
@@ -32,9 +32,21 @@ declare class sftp {
 
     get(
         path: string,
-        dst?: string | NodeJS.WritableStream,
+        dst: string,
         options?: sftp.TransferOptions,
-    ): Promise<string | NodeJS.WritableStream | Buffer>;
+    ): Promise<string>;
+
+    get(
+        path: string,
+        dst: NodeJS.WritableStream,
+        options?: sftp.TransferOptions,
+    ): Promise<NodeJS.WritableStream>;
+
+    get(
+        path: string,
+        dst?: undefined,
+        options?: sftp.TransferOptions,
+    ): Promise<Buffer>;
```

Comment:
This looks OK but does the most-general union-based signature need to be left behind at the bottom? I'm not sure what the behavior will be if you pass in a union, or use it in a callback position where it could be any of these.